.github/workflows/ubuntu-lint.yml

```diff
@@ -19,18 +19,14 @@ jobs:
           ruby-version: 2.6.5
       - name: Install Dependencies
         run: rake setup
-      - name: Run Lint
-        run: |
-          gem install rubocop -v "~>0.74.0"
-          util/rubocop
```

Comment:
Maybe keep the "Run Lint" title and run `rake rubocop` here, and we don't need the extra step running `rake rubocop:bundler`.

Reply:
Thanks, this is getting simpler each time!